### ISSUE TITLE
Error: Unsupported valibot schema: enum

### ISSUE THREAD

**Issue by philippone** [ciscoheat/sveltekit-superforms#373]
**Description**
When I try to use an enum in my schema I get the following error:

```
Error: `line` must be greater than 0 (lines start at line 1)
```

or when I change the schema when vite dev server is already running:

```
Error: Unsupported valibot schema: enum
````

Enum:
```typescript
export enum MyTestEnum {
	A = 1,
	B = 2,
	C = 3
}
```

Schema:
```typescript
import * as v from 'valibot';
const schema = v.object({
    enumField: v.enum_(MyTestEnum)
});
```
[Stackblitz with error](
https://stackblitz.com/edit/superforms-2-testing-valibot-nkqqrq?file=src%2Froutes%2Fschema.ts)

**Comment by ciscoheat** [comment]
The valibot schema converter has been updated to support this now, so it should work in the next release.

**Comment by ciscoheat** [comment]
Should be fixed now with 2.7.0, can you confirm that it is?
